pkg/helm/handlers/handlers.go

```diff
@@ -9,20 +9,28 @@ import (
 	"helm.sh/helm/v3/pkg/action"
 	"helm.sh/helm/v3/pkg/chart"
 	"helm.sh/helm/v3/pkg/release"
+	"helm.sh/helm/v3/pkg/repo"
+	"k8s.io/client-go/dynamic"
+	"k8s.io/client-go/kubernetes"
+	v1 "k8s.io/client-go/kubernetes/typed/core/v1"
+	"k8s.io/client-go/rest"
+	"sigs.k8s.io/yaml"
 
 	"github.com/openshift/console/pkg/auth"
 	"github.com/openshift/console/pkg/helm/actions"
+	"github.com/openshift/console/pkg/proxy"
 	"github.com/openshift/console/pkg/serverutils"
 )
 
 var (
 	plog = capnslog.NewPackageLogger("github.com/openshift/console", "helm")
 )
 
-func New(apiUrl string, transport http.RoundTripper) *helmHandlers {
+func New(apiUrl string, transport http.RoundTripper, chartProxy *proxy.Config, defaultRepoCACert []byte) *helmHandlers {
```

Comment:
we do not need `chartProxy` arg, not used anywhere.

Reply:
Done